[2024-02-01T22:08:12.083+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: challenger_leaderboard.tf_to_df manual__2024-02-01T22:08:08.686919+00:00 [queued]>
[2024-02-01T22:08:12.088+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: challenger_leaderboard.tf_to_df manual__2024-02-01T22:08:08.686919+00:00 [queued]>
[2024-02-01T22:08:12.088+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-01T22:08:12.098+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): tf_to_df> on 2024-02-01 22:08:08.686919+00:00
[2024-02-01T22:08:12.102+0000] {standard_task_runner.py:60} INFO - Started process 96 to run task
[2024-02-01T22:08:12.103+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'challenger_leaderboard', 'tf_to_df', 'manual__2024-02-01T22:08:08.686919+00:00', '--job-id', '252', '--raw', '--subdir', 'DAGS_FOLDER/python_tft_tier.py', '--cfg-path', '/tmp/tmpceock_fx']
[2024-02-01T22:08:12.104+0000] {standard_task_runner.py:88} INFO - Job 252: Subtask tf_to_df
[2024-02-01T22:08:12.167+0000] {task_command.py:423} INFO - Running <TaskInstance: challenger_leaderboard.tf_to_df manual__2024-02-01T22:08:08.686919+00:00 [running]> on host 964619f82b50
[2024-02-01T22:08:12.270+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='challenger_leaderboard' AIRFLOW_CTX_TASK_ID='tf_to_df' AIRFLOW_CTX_EXECUTION_DATE='2024-02-01T22:08:08.686919+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-01T22:08:08.686919+00:00'
[2024-02-01T22:08:12.351+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-01T22:08:12.370+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=challenger_leaderboard, task_id=tf_to_df, execution_date=20240201T220808, start_date=20240201T220812, end_date=20240201T220812
[2024-02-01T22:08:12.396+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-01T22:08:12.427+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
